schema/repos.json

```
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://ocr-d.de/repos.schema.json",
+    "title": "Repository Information",
+    "description": "Information needed on projects for the OCR-D Kwalitee Dashboard",
+    "type": "object",
+    "properties":{
+        "additional_info": {
```

This should be a JSON-Schema object as well, i.e.

```suggestion
        "additional_info": {
          "type": "object",
          "properties": { ...
```

---

You're right, thanks for pointing that out!